licenses.json

```diff
@@ -8774,7 +8900,7 @@
     "publisher": "Web SDK Customer Support",
     "email": "web-sdk@onfido.com",
     "url": "https://github.com/onfido",
-    "path": "/home/ethanify/Work/onfido-sdk-ui",
+    "path": "/Users/stefaniacardenas/Desktop/Code/onfido/onfido-sdk-ui",
```

Comment:
Out of scope for this PR's work, but only just noticed this local dev directory path overwrite happening here. 😬